frontend/src/pages/dashboard/components/sidebar/sidebar.tsx

```diff
@@ -31,15 +31,21 @@ const Sidebar: React.FC<Properties> = ({
     >
       <ul className={styles.list}>
         {TABS.map((tab) => (
-          <li key={tab.name}>
+          <li
+            className={
+              selectedTab === tab.name
+                ? getValidClassNames(styles.item, styles.active)
+                : getValidClassNames(styles.item)
+            }
+            key={tab.name}
+          >
```

Comment:
const isSelectedActiveTab = selectedTab === tab.name;

```suggestion
          <li
            className={ getValidClassNames(styles.item, isSelectedActiveTab && styles.active)}
            key={tab.name}
          >
```